packages/nocodb/src/db/BaseModelSqlv2.ts

```diff
@@ -4512,6 +4514,24 @@ class BaseModelSqlv2 {
     return true;
   }
 
+  /*
+   *  Utility method to validate database constraints
+   */
+  protected validateConstraints(
+    column: Column<any>,
+    data: Record<string, any>,
+  ) {
+    if (
+      typeof data[column.title] === 'string' &&
+      column.dtxp &&
```

Comment:
for `enum` type in MySQL this dtxp value will be comma separated string value so we have to skip that case as well